### PR TITLE
github: Enable dependabot

### PR DESCRIPTION
**Pull request by sameo**
For automatic tracking of dependencies and version updates.

### CODE REVIEW

**Review by Xynnn007 — APPROVED** [review]
lgtm, thanks!

**Review by Xynnn007 — COMMENTED** [review]
We also need to track go libs because cgo is used
```yaml
# Dependencies listed in go.mod
  - package-ecosystem: "gomod"
    directory: "/attestation-service/src/cgo" # Location of shim's go.mod
    schedule:
      interval: "daily"
    open-pull-requests-limit: 1
```

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: cargo
```
Comment (Xynnn007) [inline .github/dependabot.yml:3]: Do we need an indent here? As https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#example-of-a-basic-setup-for-three-package-managers shows, the whole `package-ecosystem` object should be one of the list of values ​​of `updates`.

Comment (sameo) [reply .github/dependabot.yml]: Fixed now. And I also added the go dependency, thanks for catching this @Xynnn007

Comment (dubek) [reply .github/dependabot.yml]: I see in that link that there's also tracking for github actions:

```yaml
  # Maintain dependencies for GitHub Actions
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "weekly"
```

should we add that as well?

Comment (sameo) [reply .github/dependabot.yml]: Good suggestion, done now. Thanks.

**Pull request merged**